Functions/xASL_quant_Basil.m

```diff
@@ -46,28 +46,35 @@
     end
     fprintf('%s\n', 'Note that any file not found warnings can be ignored, this pertains to the use of symbolic links by BASIL');
     
+    % Remove residual BASIL-related files
     xASL_delete(pathBasilOptions);
     xASL_delete(pathBasilInput);
-    xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^.*basil.*$', 1, [0 Inf]); % remove residual BASIL-related files
+    xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^.*basil.*$', 1, [0 Inf]);
     
     %% 3. Write the PWI as Nifti file for Basil to read as input
-    % FIXME would be good to have a brain mask at this point
-    % PM: if this would be a brainmask as well, we can skip creating a
-    % dummy input image here
+    % FIXME would be good to have a brain mask at this point -> PM: if this would be a brainmask as well, we can skip creating a dummy input image here
     PWI(isnan(PWI)) = 0;
     
-    xASL_io_SaveNifti(x.P.Path_PWI, pathBasilInput, PWI, [], 0);
+    if size(PWI,4) == 1 
```

Comment:
@MDijsselhof I believe BASIL always requires the PWI4D, because even with single PLD, it can then still do some modelling. Also, we should check if PWI4D exists if you have a single PLD scan with only 1 control-label subtraction (it should). Let's make sure that PWI4D always exists, even if it is the same as PWI.